modules/ROOT/pages/uploadcare.adoc

```diff
@@ -0,0 +1,206 @@
+= {pluginname} plugin
+:navtitle: Image Optimizer Powered by Uploadcare
+:description: The Image Optimizer Powered by Uploadcare plugin allows you to optimize images in your content.
+:description_short: 
+:plugincode: uploadcare
+:pluginname: Image Optimizer Powered by Uploadcare
+:keywords: plugin, {plugincode}, image, align, transform, alt text, caption, adjust, filter
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-7.6v.adoc[]
+
+== Overview
+
+The **Image Optimizer** plugin leverages a suite of powerful features for hosting, serving and edit images like responsive image delivery, automatic image file format selection, automatic compression and non-destructive image transformations and adjustments, all powered by link:https://uploadcare.com[Uploadcare].
+
+== Key benefits
+
+* Improve page loading speed and save on bandwidth by serving images in the optimal size, format, and compression based on the visitor's browser and device.
+* Ditch the image editor and adjust images right inside {productname} with non-destructive transformations and adjustments. Change your mind any time!
+* Stay safe and compliant with secure uploads, built in malware protection, and unsafe content detection
+* Enterprise-ready scalable cloud storage provided by Uploadcare, delivered through its lightning-fast global CDN, and compliant with SOC2 and GDPR standards.
+
+== Interactive example
+
+liveDemo::{plugincode}[]
+
+== Basic setup
+
+To add the {pluginname} plugin to the editor, add `{plugincode}` to the `plugins` option in the editor configuration.
+
+For example:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'uploadcare',
+  toolbar: 'uploadcare',
+  uploadcare_public_key: '<your-public-key>',
+  // uploadcare_signed_upload_auth_provider: (_publicKey) => Promise.resolve({
+  //   signature: 'sig',
+  //   expire: 123
+  // }), // Recommended option for secure uploads
+});
+----
+
+[NOTE]
+The {pluginname} plugin overrides the xref:quickbars.adoc[Quickbar] plugin. To ensure a better user experience, avoid using xref:quickbars.adoc[Quickbar] when using the {pluginname} plugin.
```

Comment:
Not sure about the English here but they can use `uploadcare` with `quickbars` it will just at the moment of writing produce two `image` buttons since both plugins provide the same functionallity however we have overriden the `quickimage` to insert a UC placeholder.

```suggestion
The {pluginname} plugin overrides the xref:quickbars.adoc[Quickbar] quickimage toolbar item. To ensure a better user experience and to avoid having two image buttons configure `quickbars_insert_toolbar` to omit the `quickimage` toolbar item.
```